docs/src/manual/solutions.md

```diff
@@ -489,3 +489,58 @@ for i in 2:result_count(model)
     end
 end
 ```
+
+## Checking feasibility of solutions
+
+To check the feasibility of a primal solution, use
+[`primal_feasibility_report`](@ref), which takes a `model`, a dictionary mapping
+each variable to a primal solution value (defaults to the last solved solution),
+and a tolerance `atol` (defaults to `0.0`).
+
+The function returns a dictionary which maps the infeasible constraint
```

Comment:
.. returns a dictionary whose keys are the constraints with an infeasibility greater than the supplied tolerance `atol`. The value corresponding to each key is the respective infeasibilility. Infeasibility is defined as the distance between the primal value of the constraint (see `MOI.ConstraintPrimal`) and the nearest point by Euclidean distance in the corresponding set.